Lib/pydoc.py

```diff
@@ -75,8 +75,9 @@ class or function within a module or module in a package.  If the
 from reprlib import Repr
 from traceback import format_exception_only
 
-from _pyrepl.pager import (get_pager, plain, pipe_pager,
-                           plain_pager, tempfile_pager, tty_pager)
+from _pyrepl.pager import (get_pager, pipe_pager,
+                           plain_pager, tempfile_pager, tty_pager,
+                           plain)  # noqa
```

Comment:
Which is the import being flagged as unused here? Why does it need to stay?

Reply:
`pydoc.plain()` is an alias to `_pyrepl.pager.plain()`. It has to stay since it's part of pydoc API, and test_pydoc fails if you remove it.

`# noqa` only applies to the current line, no?

Reply:
> `pydoc.plain()` is an alias to `_pyrepl.pager.plain()`. It has to stay since it's part of pydoc API, and test_pydoc fails if you remove it.

I see, thanks. Maybe add a comment explaining that to the code?

> `# noqa` only applies to the current line, no?

It depends on the tool, the rule and the AST node. For example, in this:

```py
from collections import (  # noqa: F401
    deque,
    OrderedDict,
    ChainMap,
)
```

the `noqa` comment will cause Ruff to ignore all items being imported in that statement, because they constitute a single `ImportFrom` node in the AST and the `noqa` comment is applied on the starting line number of the AST node.

Reply:
I moved the import to a separated statement and added a comment.